testdata/p4_14_samples/parser_value_set2.p4

```diff
@@ -0,0 +1,77 @@
+
+// two consecutive parser value sets separate by a normal parse state
+
+header_type ethernet_t {
+    fields {
+        dstAddr : 48;
+        srcAddr : 48;
+        etherType : 16;
+    }
+}
+
+header_type ipv4_t {
+    fields {
+        version : 4;
+        ihl : 4;
+        diffserv : 8;
+        totalLen : 16;
+        identification : 16;
+        flags : 3;
+        fragOffset : 13;
+        ttl : 8;
+        protocol : 8;
+        hdrChecksum : 16;
+        srcAddr : 32;
+        dstAddr: 32;
+    }
+}
+
+header ethernet_t ethernet;
+header ethernet_t inner_ethernet;
+header ipv4_t ipv4;
+
+@pragma parser_value_set_size 4
+parser_value_set pvs0;
```

Comment:
Should probably have a testcase with a `pragma parser_value_set_size` other than the default size to make sure its getting set up properly...